server/command.go

```diff
@@ -159,11 +159,16 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User, topi
 	createMeetingWithPMI := false
 	switch userPMISettingPref {
 	case zoomPMISettingValueAsk:
-		p.askPreferenceForMeeting(user.Id, args.ChannelId)
+		p.askPreferenceForMeeting(user.Id, args.ChannelId, args.RootId)
 		return "", nil
 	case "", trueString:
 		createMeetingWithPMI = true
 		meetingID = zoomUser.Pmi
+
+		if meetingID <= 0 {
+			p.postEphemeral(user.Id, args.ChannelId, args.RootId, "To use Personal Meeting ID (PMI) for creating the meeting, you need to `Enable Personal Meeting ID` from your [zoom settings](https://zoom.us/profile/setting).")
+			return "", nil
+		}
```

Comment:
Does the user need to reconnect their account after changing this setting?

Reply:
@mickmister No, we are fetching the user using Zoom client every time we create a meeting. So, we always have the latest data.